connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverterConfig.java

```diff
@@ -40,6 +40,11 @@ public class JsonConverterConfig extends ConverterConfig {
     private static final String SCHEMAS_CACHE_SIZE_DOC = "The maximum number of schemas that can be cached in this converter instance.";
     private static final String SCHEMAS_CACHE_SIZE_DISPLAY = "Schema Cache Size";
 
+    public static final String SCHEMA_FILE_CONTENT_CONFIG = "schema.content";
```

Comment:
```suggestion
    public static final String SCHEMA_CONTENT_CONFIG = "schema.content";
```

The references to "FILE" doesn't seem relevant as there are no files involved.